frontend/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const path = 'path'
+const webpack = 'webpack'
+
+const config = module.exports = {
+  context: __dirname,
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loaders: [
+          "babel-loader?cacheDirectory"
+        ]
+      }
+    ]
+  },
+  entry: {
+    'metamaps.bundle': './src/index.js'
+  },
+  output: {
+    path: '../app/assets/javascripts/webpacked',
+    filename: '[name].js'
+  }
+}
```

Comment:
This file just basically sets things up so running `webpack` builds `src/index.js` and all imported modules into a bundle at `../app/assets/javascripts/webpacked/metamaps.bundle.js`